src/main/java/forceitembattle/commands/CommandStart.java

```diff
@@ -90,11 +91,15 @@ private void startGame(Integer joker) {
             player.setHealth(20);
             player.setSaturation(20);
             player.getInventory().clear();
-            player.getInventory().setItem(4, new ItemBuilder(Material.BARRIER).setAmount(joker).setDisplayName("§5Skip").getItemStack());
+            player.getInventory().setItem(4, new ItemBuilder(Material.BARRIER).setAmount(joker).setDisplayName("§8» §5Skip").getItemStack());
+
+            player.getInventory().addItem(new ItemStack(Material.STONE_AXE));
+            player.getInventory().addItem(new ItemStack(Material.STONE_PICKAXE));
+
             player.setLevel(0);
             player.setExp(0);
             player.setWalkSpeed(0.2f);
-            player.setStatistic(Statistic.TIME_SINCE_REST, 3);
+            player.setStatistic(Statistic.TIME_SINCE_REST, 72000); // 1hr = 3600 seconds * 20 ticks

```

Comment:
What is that good for?

Reply:
Phantoms spawn if the "Time Since Last Rest" is at least 1 hour (72000 ticks or 3 in-game days)

Reply:
aah I see and phantoms are needed for items. smart.